extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -294,14 +299,33 @@ private function getHeroImageUrl() {
 		return $heroImageUrl;
 	}
 
-	private function calculateCurrentUserRank( $userContributionCount , $topContributors ) {
+	private function getBenefitsModalImageUrl() {
+		$url = '';
+		// we need variable to pass it by reference to helper
+		$title = self::COMMUNITY_PAGE_BENEFITS_MODAL_IMAGE;
+		$modalFile = WikiaFileHelper::getFileFromTitle( $title );
+		if ( $modalFile ) {
+			$ratio = floatval( $modalFile->getWidth() ) / floatval( $modalFile->getHeight() );
```

Comment:
You may have division by zero in case getHeight() return 0 -> i.e. file without height, like OGG